app/views/schools/contacts/edit.html.erb

```diff
@@ -2,21 +2,21 @@
 
 <% if @contact.user == current_user %>
   <h2>
-    Update contact details for email or text alerts
+    <%= t('schools.contacts.edit.update_contact_details_for_email_or_text_alerts') %>
   </h2>
   <%= render 'shared/contact_explainer' %>
 <% else %>
   <h2>
-    Update contact details <%= "for #{@contact.user.name}" if @contact.user %>
+    <%= t('schools.contacts.edit.update_contact_details_for', user_name: @contact.user.name) if @contact.user %>
```

Comment:
I think this used to append "for #{@contact.user.name}" if a @contact.user is present. Not sure the logic you have replaced it with is the same - looks like it won't display the whole line now if there is no @contact.user? Is this right?

Reply:
oops yep!  good spot.  i'll fix that.   Thanks @urbanwide

Reply:
hi @urbanwide I've pushed a fix for this and the same in new.html.erb.  would you mind giving another quick review before i merge.  thanks for spotting this :)